tests/options.js

```diff
@@ -38,36 +38,30 @@ define(function () {
                     ]
                 }
             ],
-        },
+        }
+    ];
+
+    var fixtures = [
         {
             name: "Some Fixture",
-            sourceUri: "jr://fixture/some-fixture",
-            defaultId: "somefixture",
-            rootNodeName: "foos",
-            levels: [
-                {
-                    nodeName: "foo",
-                    subsets: [
-                        {
-                            name: "woos",
-                            // should handle quotes
-                            selector: "@foo_type=\"woo\""
-                        }
-                    ]
-                },
-                {
-                    nodeName: "bar",
-                    subsets: [
-                        {
-                            name: "eggs",
-                            selector: "@bar_type='eggs'"
-                        }
-                    ]
-                }
+            sourceUri: "jr://fixture/item-list:some-fixture",
+            columns: [
+                'column1',
+                'column2',
+                'column3'
+            ]
```

Comment:
Is this what a reasonable fixture description will look like, or will this be fleshed out more later? I think we'll need to be able to model somewhat arbitrary XML structure eventually. Something like
```js
{
  sourceUri: "jr://fixture/commtrack:products",
  structure: { // the "structure" key is always present
    products: {
      product: {
        "@id": {},  // attribute
        name: {},   // child element with no attributes and no child nodes
        price: {},
        etc: {
          could: {have: {}},
          more: {structure: {"@in": {}, "@here": {}}}
        }
      }
    }
  }
}
```

Of course there are still XML documents that cannot be modeled with this convention, but I'm not sure we care.